ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -623,6 +628,33 @@ def osd_encryption_verification():
             )
             raise ValueError("OSD is not encrypted")
 
+    if ocs_version > version.VERSION_4_6:
```

Comment:
We are skipping the encryption verification for < 4.6 versions here[1]. 
This condition can be removed,

[1] https://github.com/red-hat-storage/ocs-ci/pull/5555/files#diff-193776bad25880fee96c0b8075161c63d5172b2d518d974326c937672cf7dd1fR604-R605

Reply:
@PrasadDesala  but that condition includes 4.6 but luks header info feature isn't backported to 4.6.. so I want to skip even for 4.6

Reply:
ack